resources/views/layouts/footer.blade.php

```diff
@@ -26,6 +26,8 @@
             <div class="switcher grow:2">
                 <div class="stack" id="contact" tabindex="-1">
                     <h2>{{ __('Contact') }}</h2>
+                    <x-interpretation class="interpretation--start" name="{{ __('Contact', [], 'en') }}"
+                        namespace="contact" />
```

Comment:
May need to modify this name so that it is distinguishable from other contact information. Maybe something like "Site Contact" or "Footer: Contact", etc.